packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -150,7 +152,37 @@ enum TraversalEdgeBehavior {
 abstract class FocusTraversalPolicy with Diagnosticable {
   /// Abstract const constructor. This constructor enables subclasses to provide
   /// const constructors so that they can be used in const expressions.
-  const FocusTraversalPolicy();
+  ///
+  /// {@template flutter.widgets.FocusTraversalPolicy.requestFocusCallback}
+  /// The `requestFocusCallback` can be used to override the default behaviour
```

Comment:
Please prefer US english. See: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#spell-words-in-identifiers-and-comments-correctly

```suggestion
  /// The `requestFocusCallback` can be used to override the default behavior
```

Reply:
Thanks for the heads-up, English is not my first language and honestly sometimes the nuances between UK and US English go over my head.